nasa/ow_europa
# Render different material for grinder tailings.

To test:
* roslaunch ow atacama_y1a.launch
* rosrun ow_lander task_grind.py -x 1.75 -y 0 -z -0.25
* rosrun ow_lander pan_tilt_move_joints.py 1.2 0.8
* rosrun ow_lander camera_capture.py

The resulting image looks decent, but I'm not a big fan of the look of the new material. If you zoom in closely in gzclient it looks a lot more like shiny stucco than snow. I'll see if I can do a separate ticket to make things look better, maybe by adding some simple subsurface scattering technique.
